packages/grid/x-data-grid/src/tests/layout.DataGrid.test.tsx

```diff
@@ -59,14 +59,10 @@ describe('<DataGrid /> - Layout & Warnings', () => {
     });
 
     it('should resize the width of the columns', async () => {
-      // Using a fake clock also affects `requestAnimationFrame`.
-      // Calling clock.tick() should call the callback passed, but it doesn't work.
-      stub(window, 'requestAnimationFrame').callsFake((fn: any) => fn());
-      stub(window, 'cancelAnimationFrame');
```

Comment:
Mocking `requestAnimationFrame` doesn't feel right so I inverted the logic: use real clock by default and fake clock on demand.